portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -57,13 +58,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       setMaxComment(request, request.getParameter(MAXCOMMENT_PARAMETER));
     }
 
-    response.sendRedirect(COMMENT_HTML_PARAMETER);
+    response.sendRedirect(this.COMMENT_HTML_PARAMETER);
```

Comment:
So while `this` can be used here, I think we need to be consistent (i.e. use `this` for all member variable access or never use it)